app/controllers/application_controller.rb

```diff
@@ -149,8 +149,8 @@ def clear_session_credentials
 
   def render_exception(exception)
     # In development or the admin interface let Rails handle the exception
-    # with its stack trace templates
-    if Rails.application.config.consider_all_requests_local || show_rails_exceptions?
+    # unless it's for the favicon as this breaks the REPL in better_errors
+    if (Rails.application.config.consider_all_requests_local || show_rails_exceptions?) && request.path != "/favicon.ico"
```

Comment:
>Could we just add the favicon to the admin layout?

Nope :frowning: It seems the browser still tries to fetch `/favicon.ico`, which results in Alaveteli throwing an exception and thereby breaking the better_errors REPL.